tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es2017",
     "lib": ["dom", "dom.iterable", "es2017", "esnext.asynciterable"],
-    "allowJs": false,
+    "allowJs": true,
```

Comment:
Why allowJS?

Reply:
so that js files in the `src` folder could be linted instead of just reporting an error... such as `setup.ssr.js` and future things...